modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/types/ConsensusRounds.scala

```diff
@@ -0,0 +1,15 @@
+package org.tessellation.sdk.domain.healthcheck.types
+
+import org.tessellation.sdk.domain.healthcheck.HealthCheckRound
+import org.tessellation.sdk.domain.healthcheck.types.ConsensusRounds.InProgress
+
+final case class ConsensusRounds[F[_]](
+  historical: List[HistoricalRound],
+  inProgress: InProgress[F]
+)
+
+object ConsensusRounds {
+  private type Rounds[F[_]] = Map[HealthCheckKey, HealthCheckRound[F]]
+  type InProgress[F[_]] = Rounds[F]
```

Comment:
Shouldn't that be a `sealed trait` with two classes for `InProgress` and `Finished` instead of the two type aliases?

Reply:
no, I need both these types separately, e.g.:
```scala
(rounds: ConsensusRounds.InProgress[F])
...
F[(ConsensusRounds.Finished[F], ConsensusRounds.InProgress[F])]
```